Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/07-SyncUpDetailNavigation/SyncUpDetailNavigation-01-code-0001.swift

```diff
@@ -2,7 +2,7 @@ import ComposableArchitecture
 import SwiftUI
 
 @Reducer
-struct App {
+struct AppFeature {
```

Comment:
To avoid conflicting with the `SwiftUI.App` protocol, I switched mention of this to `AppFeature`. There were a couple times where this was referenced as `AppFeature` in the tutorial anyways, so it seemed like the right name.